frontend/src/components/TagInput/TagInput.tsx

```diff
@@ -71,6 +71,7 @@ const TagInput = ({ value, handleValue, resetValue, tags, setTags }: Props) => {
         {tags?.map((tag, idx) => (
           <TagButton
             key={idx}
+            id={idx}
```

Comment:
```suggestion
            id={tag.id ?? 0} 
```
이런식으로 주면 tag.id 가 있는 경우에는 색깔이 입혀지고, 없는 경우에는 첫번째 색상으로 통일 될 것 같아요!

Reply:
이를 위해서는 TagInput에서 props로 태그명 문자열 배열을 받고 있는데, 이를 태그 객체 배열을 받도록 변경해야 할 것 같네용